### PR TITLE
Feature/gh 1232 tooltip

### PR DESCRIPTION
**Pull request by DmitryBespalov**
- Extension of the white tooltip
- Added showing green tooltip where it is needed by specification

### CODE REVIEW

File: SafeUIKit/SafeAppUI/MainFlow/Tokens/ManageTokens/ManageTokensTableViewController.swift [hunk 1]
```diff
@@ -59,14 +61,19 @@ final class ManageTokensTableViewController: UITableViewController {
             Timer.wait(delay)
             self.tokens = ApplicationServiceRegistry.walletService.visibleTokens(withEth: false)
             DispatchQueue.main.async {
-                self.tableView.reloadData()
+            self.tableView.reloadData()
```
Comment (sche) [inline SafeUIKit/SafeAppUI/MainFlow/Tokens/ManageTokens/ManageTokensTableViewController.swift]: did you shift it by a mistake?

File: SafeUIKit/SafeUIKit/Components/FeeCalculation/FeeCalculationAssetLine.swift [hunk 2]
```diff
@@ -109,6 +109,8 @@ public class FeeCalculationAssetLine: FeeCalculationLine {
         }
     }
 
+    public static let settingsButtonTag = 0x10
```
Comment (sche) [inline SafeUIKit/SafeUIKit/Components/FeeCalculation/FeeCalculationAssetLine.swift:112]: funny choice of a number for tag :)

**Review by sche — APPROVED** [review]
👍

File: SafeUIKit/SafeAppUI/MainFlow/Tokens/ManageTokens/ManageTokensTableViewController.swift (hunk 1 above)
Comment (DmitryBespalov) [reply SafeUIKit/SafeAppUI/MainFlow/Tokens/ManageTokens/ManageTokensTableViewController.swift]: yes

File: SafeUIKit/SafeUIKit/Components/FeeCalculation/FeeCalculationAssetLine.swift (hunk 2 above)
Comment (DmitryBespalov) [reply SafeUIKit/SafeUIKit/Components/FeeCalculation/FeeCalculationAssetLine.swift:112]: ✨✨✨magic✨✨✨

**Pull request merged**